src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java

```diff
@@ -133,4 +155,16 @@ public Entity scheduledInterviewToEntityForUpdate(ScheduledInterview scheduledIn
     scheduledInterviewEntity.setProperty("interviewee", scheduledInterview.intervieweeEmail());
     return scheduledInterviewEntity;
   }
+
+  private List<Entity> getEntitiesInRange(long minTime, long maxTime, Optional<Filter> filterOpt) {
+    Filter minFilter =
+        new FilterPredicate("startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime);
+    Filter maxFilter = new FilterPredicate("startTime", FilterOperator.LESS_THAN_OR_EQUAL, maxTime);
```

Comment:
I think the name here should be "endTime"?

Reply:
Datastore only supports one inequality filter per query. To work around Meagan suggested setting the max filter to be at most 60 minutes before the maxTime.

Reply:
Please add this as a comment for future readers.

Reply:
Please add this as a comment for future readers.

Reply:
Done